src/components/elements/dropdown-search.tsx

```diff
@@ -1,12 +1,18 @@
 import React, { useEffect, useState } from 'react'
 import styled, { css } from 'styled-components'
-import PropTypes from 'prop-types'
-import { Arrow, BottomLabel, DropdownContainer, ItemList, StyledLabel } from './dropdown'
+import {
+    Arrow,
+    BottomLabel,
+    DropdownContainer,
+    DropdownProps,
+    ItemList,
+    StyledLabel,
+} from './dropdown'
 import { useDropdown } from 'components/hooks/use-dropdown'
 import device from 'themes/device'
 import { Flex } from 'components/containers'
 
-const DropdownInput = styled.input`
+const DropdownInput = styled.input<any>`
```

Comment:
Put any because of this:
<img width="399" alt="Screenshot 2022-08-03 at 08 12 01" src="https://user-images.githubusercontent.com/103182473/182590114-07256cf1-d032-4e5f-a560-952efd5c3a60.png">

Reply:
@NikitK-deriv which error do you get here?